<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSettingDlg</class>
 <widget class="QDialog" name="ImageSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>476</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>476</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImageSettingDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetTitle" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetTitle{
background-color:rgb(40, 40, 40);
border-top-right-radius:4px;	
border-top-left-radius:4px;	
}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayoutTitle"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2{
background-color:rgb(50, 50, 50);
border-top-right-radius:4px;	
border-top-left-radius:4px;	
}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="widget_9" native="true">
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>430</width>
             <height>325</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>430</width>
             <height>325</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(28, 28, 28)</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QWidget" name="widget_5" native="true">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>275</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>275</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="1">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>123</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="0">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>186</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="1" rowspan="2">
                <layout class="QGridLayout" name="gridLayoutCenter">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                </layout>
               </item>
               <item row="3" column="1">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>123</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="2">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>170</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QWidget" name="widget_4" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>40</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="styleSheet">
            <string notr="true">QWidget#widget{
background-color: rgb(50, 50, 50);
}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>23</number>
            </property>
            <property name="rightMargin">
             <number>23</number>
            </property>
            <item>
             <widget class="ComboBox" name="comboBoxPos">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <item>
               <property name="text">
                <string>左上角</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>左下角</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>中上位置</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>中下位置</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>右上角</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>右下角</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>全屏</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnChangeImage">
              <property name="styleSheet">
               <string notr="true">color:rgb(98,181,215);
text-decoration: underline;</string>
              </property>
              <property name="text">
               <string>更换图片</string>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget#widget_8{
border-bottom-right-radius:4px;	
border-bottom-left-radius:4px;	
}
</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="saveBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#saveBtn:hover{background-color: rgb(255, 51, 51);border-radius: 12px;color: rgb(255, 235, 235);font: 9pt &quot;微软雅黑&quot;;}
QPushButton#saveBtn{background-color: rgba(255, 45, 52, 51);border: 1px solid rgb(255, 51, 51);border-radius: 12px;color: rgb(208, 202, 203);font: 9pt &quot;微软雅黑&quot;;}</string>
              </property>
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
